### PR TITLE
Upgrade grafana version to the latest

### PR DESCRIPTION
**Pull request by gecube**
Screenshot 2024-09-23 at 11 45 16

### CODE REVIEW

**Comment by gecube** [comment]
everything works like with old version.

**Comment by gecube** [comment]
also fixed the paths for predictor panels.

File: docker-compose.yml [hunk 1]
```diff
@@ -26,14 +26,14 @@ services:
...
-    image: grafana/grafana:9.3.4
+    image: grafana/grafana:11.2.0-ubuntu
     container_name: openstef-grafana
     ports:
       - 3000:3000
     environment:
       GF_PANELS_DISABLE_SANITIZE_HTML: "true"
   grafana-configurator:
-    image: alpine:3.7
+    image: alpine:3.20.3
```
Comment (JanMaartenvanDoorn) [inline docker-compose.yml:36]: Great to update these!

File: grafana-configurator/volumes/config/configurator.sh [hunk 2]
```diff
@@ -16,10 +26,14 @@ done
...
 # Wraps the dashboard in a JSON message and posts it to the API:
 # https://grafana.com/docs/grafana/latest/http_api/dashboard/
-for i in ${BASEDIR}/dashboards/*.json; do \
-    json=`cat ${i}`
-    curl -X "POST" "http://grafana:3000/api/dashboards/db" \
-        -H "Content-Type: application/json" \
-        --user $GRAFANA_USER:$GRAFANA_PASSWORD \
-        --data "{\"dashboard\": ${json}, \"folderId\": 0, \"message\": \"Inital upload\", \"overwrite\": false}"
-done
+#for i in ${BASEDIR}/dashboards/*.json; do \
```
[the remark this answers is not in the document]
Comment (gecube) [reply grafana-configurator/volumes/config/configurator.sh]: @JanMaartenvanDoorn thanks, fixed.

**Review by JanMaartenvanDoorn — APPROVED** [review]

**Pull request merged**